enterprise/frontend/src/metabase-enterprise/api/ai-streaming/requests.ts

```diff
@@ -55,7 +55,7 @@ export async function aiStreamingQuery(
     });
 
     if (!response.ok) {
-      throw new Error(response.statusText);
+      throw new Error(`Response status: ${response.status}`);
```

Comment:
this was causing issues with being able to match on the value for some weird reason... stringifying it helps 🤷